nni/algorithms/compression/tensorflow/pruning/one_shot.py

```diff
@@ -3,15 +3,15 @@
 from nni.compression.tensorflow import Pruner
 
 __all__ = [
-    'OneshotPruner',
```

Comment:
Maybe we can keep this for user who wants to write their own pruner?

Reply:
If we export it than it's a public API and we need to maintain backward compatibility.
I think it's an internal helper and don't want to make it a formal API.
If a user really want it, they should copy the code.